src/components/SingleArticle.jsx

```diff
@@ -19,7 +21,7 @@ const SingleArticle = () => {
 
   const navigate = useNavigate();
   const clickHandlerHome = () => {
-    navigate("/");
+    navigate(-1);
```

Comment:
I haven't used useNavigate myself so I'm not sure exactly how it works but at the moment you have to click the Back To All News Articles button twice to get back, the first time you click it nothing happens. If you just want this to take you to the page listing all articles then it may be worth just having it be a `<Link></Link>`to `'/'`